### DISCUSSION TITLE
IEP 9999: Apply welcome discussion

### DISCUSSION THREAD

**Discussion by DyrellC** [DyrellC/IEPs#9] [Show and tell]
### Referenced Discussion
#1 

## About 
Informational IEPs are meant to be educational and serve as a knowledge source for members of the ecosystem.  
  
## Material Type 
Not to be confused with IEP Type. This should outline if the submitted Informational IEP is a Case Studies, Publication, and After Action report, Regulatory Reports, relevant climate Policy, External Standards Reports or Treaties

## Abstract
A brief description of the change that is being proposed or the information that is being submitted as a relevant source and reference.

## Rationale 
A breakdown of the rationale as to why this proposal is being suggested, including any metrics, papers, examples or discussion material leading to the proposal creation

## Specification 
Technical details of the proposed changes. For example an API change should have the appropriate breakdown of required variables, types and return values. 

## Other Concerns
References to related and previous IEP's and discussions, or will be affected by this change. Security concerns or implications should be expanded on in this section as well. 
  
## Footnotes
A collection of footnotes cited in the IEP, and a place to list non-inline hyperlink targets.
  
## Copyright / License
Each new IEP must be placed under a dual license using an open source public domain [Open Source Approved License](https://opensource.org/licenses) and [CC0-1.0-Universal](https://choosealicense.com/licenses/cc0-1.0/)

**Comment by DyrellC** [comment]
Behold another discussion

**Reply by MatYarger** [reply]
Huzzah! Great success.

**Comment by DyrellC** [comment]
So much discussion

**Reply by MatYarger** [reply]
Indeed. It is a glory to behold.

**Comment by DyrellC** [comment]
Can't get enough of that sweet sweet discussion

**Reply by MatYarger** [reply]
For the greater good, we must continue to discuss this.
